Website/Views/WeBlog/TagCloud.cshtml

```diff
@@ -1,13 +1,33 @@
-﻿@using Sitecore.Modules.WeBlog.Extensions
+﻿@using System.ComponentModel
+@using System.Web
+@using Sitecore.Modules.WeBlog.Extensions
 @using Sitecore.Modules.WeBlog.Globalization
-@model IEnumerable<Sitecore.Modules.WeBlog.Model.TagCloudRenderingModel>
+@model Sitecore.Modules.WeBlog.Model.TagCloudRenderingModel
 
 <div class="wb-tag-cloud wb-panel">
     <h3>@Translator.Render("TAGCLOUD").ToHtmlString()</h3>
+
+    <ul class="wb-tag-sorting">
```

Comment:
Could we add a parameter to the rendering to allow the user to select if they want to show the sorting options to the user or just use a single selected option?